Patch-Source: https://gitlab.gnome.org/GNOME/libxml2/-/commit/95939d6ea3718c458620eeda850add549cd07e99
--
From 95939d6ea3718c458620eeda850add549cd07e99 Mon Sep 17 00:00:00 2001
From: "Jan Alexander Steffens (heftig)" <heftig@archlinux.org>
Date: Sat, 15 Jun 2024 00:53:15 +0200
Subject: [PATCH] xinclude: Add missing include

Gambas fails to compile with

    error: 'xmlStructuredErrorFunc' has not been declared
---
 include/libxml/xinclude.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/libxml/xinclude.h b/include/libxml/xinclude.h
index 6a67953eb..71fa4c20d 100644
--- a/include/libxml/xinclude.h
+++ b/include/libxml/xinclude.h
@@ -14,6 +14,7 @@
 #define __XML_XINCLUDE_H__
 
 #include <libxml/xmlversion.h>
+#include <libxml/xmlerror.h>
 #include <libxml/tree.h>
 
 #ifdef LIBXML_XINCLUDE_ENABLED
-- 
GitLab

